scripts/staging/validate.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# --- Default Variables ---
+CHART_VALIDATE="false"
+TRIGGER=""
+TAG=""
+
+# --- Parse arguments --- 
+for ((i=1; i <= $#; i++)); do
+  case "${!i}" in
+    --chart)
+      next=$((i+1))
+      if [[ $next -le $# ]]; then
+        CHART_VALIDATE="${!next}"
+      fi
+      ;;
+  esac
+done
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --trigger|--type)
+            TRIGGER="$2"; shift 2 ;;
+        --tag)
+            TAG="$2"; shift 2 ;;
+        --chart)
+            CHART_VALIDATE="$2"; shift 2 ;;
+        -h|--help)
+            cat <<EOF
+Usage: $0 --trigger <trigger> [--tag <tag>] [--chart <true|false>]
+Options:
+  --trigger <type>        release, staging, develop, prerelease
+  --type <type>           Alias for --trigger
+  --tag <tag>             Release tag (e.g., v2.9.0)
+  --chart <true|false>    Enable or disable Helm chart validation (default: false)
+EOF
+            exit 0 ;;
+        *) log_fatal "Unknown option $1" ;;
+    esac
+done
+
+# --- Determine paths based on CHART_VALIDATE ---
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+source "${SCRIPT_DIR}/../utils/log.sh"
+
+if [[ "${CHART_VALIDATE}" == "true" ]]; then
+    ROOT_DIR="${SCRIPT_DIR}/../../../../../../"
+    : "${PARENT_ROOT_DIR:=$ROOT_DIR}"
+else
+    ROOT_DIR="${SCRIPT_DIR}/../../../../"
+    : "${PARENT_ROOT_DIR:=$ROOT_DIR}"
+fi
+
+export SOURCE_REL="$PARENT_ROOT_DIR/utils/dependencies/scripts/release.sh"
+export NO_RUN=true 
+. "$PARENT_ROOT_DIR/scripts/release.sh"
+
+# # --- Run release.sh in a subshell ---
+# (
+#   export NO_RUN=true
+#   export CI=1
+#   echo "Running release.sh from: $PARENT_ROOT_DIR/scripts/release.sh"
+#   . "$PARENT_ROOT_DIR/scripts/release.sh"
+# )
+
```

Comment:
Remove this

```suggestion
```

Reply:
I guess you are looking at older commit , I had already removed it with last commit.